status-im/status-react
# feature #3976 - getting token transfer events for erc20 tx history

fixes #3976

### Summary:

### Testing notes (optional):

- Getting transfer events works only for networks with upstream RPC. This is a known
problem we currently have with LES.
- We don't have the complete data in case that current app is the recipient of the transfer transaction. Specifically data points related to gas are missing. Solving this limitation is in the scope of #1762 

### Steps to test:
- Open Status 
- Get some tokens, e.g. on Ropsten get them from Andrey's dapp.
- Send tokens to contacts
- Send tokens back from contacts
- Check if transactions appear in the history

status: ready
